lib/detail_page.dart

```diff
@@ -17,7 +17,7 @@ class _DetailPageState extends State<DetailPage> {
           Hero(
             tag: "hero_tag",
             child: FlutterLogo(
-              size: 200.0,
+              size: 250.0,
```

Comment:
You have updated the icon size to 250 just for one list item. Please update this for all the list icons present in this file.
```suggestion
              size: 250.0,
```